### DISCUSSION TITLE
Control datastreams to different Search Nodes

### DISCUSSION THREAD

**Discussion by ByteAndBits** [Security-Onion-Solutions/securityonion#11326] [2.4]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

64

### RAM

256GB

### Storage for /

Raid5 

### Storage for /nsm

Raid10

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Hi,

is it possible to control the datastreams from Nodes ans agents to elastic databases?

We have geo redudant datacenters. To push all alerts and logs thru VPN or MPLS will be too haevy.
So our idea: put all nodes and one or two Storage Nodes to each datacenter and stream all data to this databases localy. So only the searching runns over the tunnels. 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
That's kinda the idea for Heavy Nodes, but please note the warnings that we don't recommend them for most users:
https://docs.securityonion.net/en/2.4/architecture.html#heavy-node

Performance can be challenging due to running lots of processes on the heavy node and searching across a tunnel may not be as quick as you'd like.
